cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AuthenticationUtils.java

```diff
@@ -43,7 +46,10 @@ public class AuthenticationUtils {
     private static final String TLS_AUTH_CERT = "TLS_AUTH_CERT";
     private static final String TLS_AUTH_KEY = "TLS_AUTH_KEY";
     private static final String SASL_PASSWORD_FILE = "SASL_PASSWORD_FILE";
-    private static final String OAUTH_CONFIG = "OAUTH_CONFIG";
+    /**
+     * Key for OAuth config
+     */
+    public static final String OAUTH_CONFIG = "OAUTH_CONFIG";
```

Comment:
Does it really need to be public? I did not find any use of this outside this class.